### PR TITLE
chore(css): #FOR-684 migrate css within module

### PR DESCRIPTION
**Pull request by Gregprod33**
## Describe your changes
*Migrate css inside the module as mentioned in the documentation :
https://confluence.support-ent.fr/display/DN/Packager+du+CSS+dans+un+module*
## Checklist tests
*Make a css change like a red backround color and then buils css from the build.sh of the module and watch the changes in the browser.*
## Issue ticket number and link
[FOR-684](https://jira.support-ent.fr/browse/FOR-684)
## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: .github/workflows/dev-check-repository.yml [hunk 1]
```diff
@@ -7,17 +7,17 @@ jobs:
...
       options: --user root -v ${{ github.workspace }}:/home/node/:rw
 
     steps:
       - uses: actions/checkout@v1
       - name: Run npm install
-        run: npm install
+        run: yarn install
       - name: Run build node with Gulp
         run: node_modules/gulp/bin/gulp.js build
       - name: Run test
```
Comment (JinDouang) [inline .github/workflows/dev-check-repository.yml:19]: Il manque l'instruction pour tester le build scss

File: build.sh [hunk 2]
```diff
@@ -25,13 +25,22 @@ clean () {
...
     *)
-      docker-compose run --rm -u "$USER_UID:$GROUP_GID" node sh -c "npm install && node_modules/gulp/bin/gulp.js build"
+      docker-compose run --rm -u "$USER_UID:$GROUP_GID" node sh -c "yarn install && node_modules/gulp/bin/gulp.js build && yarn run build:sass"
   esac
 }
 
+
+buildGulp() {
+    docker-compose run --rm -u "$USER_UID:$GROUP_GID" node sh -c "node_modules/gulp/bin/gulp.js build"
+}
```
Comment (JinDouang) [inline build.sh:38]: (optionnel) 

Tu devrais rajouter 2 tâches de buildGulp pour préciser formulaire et formulaire-public

**Review by JinDouang — CHANGES_REQUESTED** [review]

File: .github/workflows/dev-check-repository.yml (hunk 1 above)
Comment (Gregprod33) [reply .github/workflows/dev-check-repository.yml:19]: J'ai ajouté les l'instruction

File: build.sh (hunk 2 above)
Comment (Gregprod33) [reply build.sh:38]: J'ai recréé 2 tâches, ça te parait ok ?

**Review by simonchaval — APPROVED** [review]

**Review by qperie — APPROVED** [review]

**Pull request merged**